code/modules/hydroponics/seed_extractor.dm

```diff
@@ -110,16 +110,22 @@
 	var/production = 0
 	var/yield = 0
 	var/potency = 0
+	var/instability = 0

```

Comment:
this or the entire class needs to be doc'd to show that it's a representation of the plant's stats.